components/calendar/src/ethiopic.rs

```diff
@@ -56,11 +54,28 @@ pub enum EthiopicEraStyle {
     /// in this calendar
     AmeteAlem,
 }
-/// The Ethiopic Calendar
```

Comment:
The docs now use both "ethiopic" and "ethiopian"

Reply:
I plan to do a full upgrade later, we should _always_ be saying ethiopian except in the BCP47 tag